girder_utils/templatetags.py

```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter()
+def lookup(value, arg):
```

Comment:
Can you add a docstring, with an explanation and sample usage?

Reply:
In particular, it'd be good to explain why the dictionary lookup provided by https://docs.djangoproject.com/en/3.1/ref/templates/language/#template-variables is insufficient.

Reply:
Are you asking for you, or just for docs? The tldr is this allows using variables in the "getter".

Reply:
I can't add a docstring without it being "101 to template tags" of `{{ mydict|mykey }}`. I added a description similar to the builtin django filters.